livekit-agents/livekit/agents/aio/channel.py

```diff
@@ -24,13 +24,6 @@ class ChanEmpty(Exception):
     pass
 
 
-def channel(
```

Comment:
Removed because there is no place to specify T so channels created in this way would need to be explicitly typed downstream which is clunkier than just create the Chan manually like
`foo_tx = foo_rx = Chan[Bar]()`